templates/base_sober.html

```diff
@@ -159,5 +172,17 @@
 		type : 'cover'
 	});
 </script>
+<script>
+{% if messages %}
+  {% for message in messages %}
+    $(function() {
+      setTimeout(function() {
+          $(".message").hide('blind', {}, 500)
+      }, 25000);
+    });
+  {% endfor %}
+{% endif %}
+</script>
+
```

Comment:
For clarity, can be moved within a templates/js/timer_flash_message.js file (or something similar) 😺